File: sampleNativeLibJv/src/main/java/com/example/samplenativelibjv/NativeLib.java
package com.example.samplenativelibjv;

public class NativeLib {

    // Used to load the 'samplenativelibjv' library on application startup.
    static {
        System.loadLibrary("samplenativelibjv");
    }

    /**
     * A native method that is implemented by the 'samplenativelibjv' native library,
     * which is packaged with this application.
     */
    public native String stringFromJNI();
}

File: sampleNativeLibKt/src/main/java/com/example/samplenativelibkt/NativeLib.kt
package com.example.samplenativelibkt

class NativeLib {

    /**
     * A native method that is implemented by the 'samplenativelibkt' native library,
     * which is packaged with this application.
     */
    external fun stringFromJNI(): String

    companion object {
        // Used to load the 'samplenativelibkt' library on application startup.
        init {
            System.loadLibrary("samplenativelibkt")
        }
    }
}